docs/rendered_changelog.txt

```diff
@@ -0,0 +1,7 @@
+Parfive 1.0.0 (2019-05-01)
```

Comment:
@vn-ki can you rename this file to have a `.rst` extension seeing how that is what it is.

Also would it not be better to be in the root of the repo as `CHANGELOG.rst`? We can still include it in the docs if we do that.